octo-core/lib/octocore/callbacks.rb

```diff
@@ -15,6 +15,9 @@ module OctoHooks
 
       # Define the after_app_init hook
       after_app_init do |args|
+        # Not sure if we need to add init to
```

Comment:
You mean, init to init be one session? But we already got that covered with the expiry limit..